source/blog/2017-02-06-finagle-loves-netty4.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: Experimental Netty 4 support in Finagle
+published: true
+post_author:
+  display_name: Vladimir Kostyukov
+  twitter: vkostyukov
+tags: Finagle, Netty4
+---
+
+It's been quite a while since [Netty 4 migration in Finagle][netty4-blog-post] was initially
+announced. We've travelled a long way and are happy to announce that there is now (as per
+[Finagle 6.42][6.42]) an experimental support for Netty 4 transports in most of the  protocols
+(Thrift, ThriftMux, Memcached, MySQL) and production-ready support for Redis.
+
+## How to enable?
+
+We encourage Finagle users to try out new Netty 4 transports for their protocols hence jump on a
+fast tract to  upcoming changes around resiliency (think of HTTP/2), performance (think of Netty 4
+pooling), and security (think of service-to-service auth).
+
+To switch the transport over to Netty 4 supply the following command line flag for a JVM process
+running Finagle.
+
+```
+-Dcom.twitter.finagle.toggle.flag.overrides=com.twitter.$protocol.UseNetty4=1.0
+```
+
+Where `$protocol` is one of the following: `mux` (use this for ThriftMux), `thrift`, `mysql`,
+`memcached`.
+
+This command line flag overrides [a feature toggle][toggles] that's evaluated at application
+start up and is *global for all clients/servers running on the same JVM instance*.
+
+Note that Netty 4 is already enabled by default in `finagle-redis` so no need for extra CLI flag
+there.
+
+## What about HTTP/1.1?
+
+HTTP/1.1 on Netty 4 is still work in progress. There are known limitations for HTTP clients, but
+we've been successfully running finagle-http servers with Netty 4 in production for several weeks.
+
+That said we encourage everyone to enable Netty 4 transport on HTTP servers running Finagle as show
+above.
+
+## What's next?
+
+We're quite optimistic about enabling Netty 4 by default in the next Finagle release for all
```

Comment:
afaik we have not set a concrete date for this. how about we give ourselves more room and say in the next 3 months barring unforeseen issues.